### DISCUSSION TITLE
Segmentation fault with "from torch_geometric.nn import GCNConv"

### DISCUSSION THREAD

**Discussion by Accelerator-thu** [pyg-team/pytorch_geometric#5523] [Q&A]
When executing `from torch_geometric.nn import GCNConv`, I got a 
`Segmentation fault (core dumped)`
And with faulthandler, results are:
<img width="969" alt="Screen Shot 2022-09-24 at 18 26 00" src="https://user-images.githubusercontent.com/87270239/192092960-3c7ef67d-d4f3-4670-a5f3-f7a417bc73b0.png">
<img width="977" alt="Screen Shot 2022-09-24 at 18 23 32" src="https://user-images.githubusercontent.com/87270239/192092858-b56428c0-9373-4bf5-b9c3-aef8d874e782.png">

How should I fix this?🤔

**Comment by Accelerator-thu** [comment]
Also, it seems that whenever I want to import something from torch_geometric.nn, there comes a Segmentation fault at the specific line.

**Comment by EdisonLeeeee** [comment]
I guess there is a mismatch between your PyTorch and external packages `torch_scatter`, `torch_sparse` ... It can be resolved by re-installing Pytorch and external packages.
